viewer/src/app/feature-service.service.ts

```diff
@@ -108,10 +143,13 @@ export class FeatureServiceService {
   constructor(private http: HttpClient) {}
 
   getFeatures(url: DataUrl): Observable<Feature<Geometry>[]> {
-    console.log(url)
+    console.log(JSON.stringify(url))
```

Comment:
dit is wel veel om te loggen. Misschien moeten we een aparte debug logger class introduceren die default uit staat maar alleen via de lokale environment.ts is ingeschakeld.

Reply:
ik maak een losse branch om te kijken of het lukt om ngx-logger (https://www.npmjs.com/package/ngx-logger)  te implemeteren voor alle logging in het gehele component